nautobot/ipam/factory.py

```diff
@@ -438,16 +438,16 @@ class Params:
         has_vrf = factory.Faker("pybool")
         is_container = factory.Faker("pybool")
         is_ipv6 = factory.Faker("pybool")
-        ipv6_cidr = factory.Faker("ipv6", network=True)
+        ipv6_cidr = UniqueFaker("ipv6", network=True)
         # faker ipv6 provider generates networks with /0 cidr, change to anything but /0
         ipv6_fixed = factory.LazyAttribute(
-            lambda o: o.ipv6_cidr.replace("/0", f"/{UniqueFaker('pyint', min_length=1, max_length=128)!s}")
+            lambda o: o.ipv6_cidr.replace("/0", f"/{faker.Faker().pyint(min_value=1, max_value=128)!s}")
```

Comment:
Also actually since `ipv6_cidr` is a network already, maybe `min_value` should be `ipaddress.IPv6Network(ipv6_cidr).prefixlen` so we only ever generate subnets of the original network rather than potentially creating supernets?

Reply:
> Also actually since `ipv6_cidr` is a network already, maybe `min_value` should be `ipaddress.IPv6Network(ipv6_cidr).prefixlen` so we only ever generate subnets of the original network rather than potentially creating supernets?

This is only replacing `/0` which is only valid for `::/0` so anything >0 will be a subset of the original network

Reply:
Thanks for clarifying my confusion! So now we have a 1/128 chance of getting something from `::/1` to `::/128` right? I wonder if we could/should instead just reject `/0` entirely and "reroll" `ipv6_cidr` until we get something that isn't /0?

Reply:
> Thanks for clarifying my confusion! So now we have a 1/128 chance of getting something from `::/1` to `::/128` right? I wonder if we could/should instead just reject `/0` entirely and "reroll" `ipv6_cidr` until we get something that isn't /0?

I added a custom faker provider in 5202e0bf014874174ef0ba5117cd2911f85a632c to fix the problem in the default ipv6 faker provider and also allow it to use integers below the ipv4 max range (`2**32`). I'm not sure what that was doing but it didn't seem necessary.